charts/k8s-monitoring/docs/RBAC.md

```diff
@@ -0,0 +1,446 @@
+# RBAC Rules for the Kubernetes Monitoring Helm Chart
+Several components have specialized RBAC rules to perform their work. This document will list the components, and their RBAC definitions.
+Several components have specialized RBAC rules to perform their work. This document will list the components, and their RBAC definitions
+## Grafana Agent
+```yaml
+- apiGroups:
+    - ''
```

Comment:
These empty entries look a bit weird

Reply:
an empty api group means that it applies to the core API group, so for pods, services, nodes, etc...